data/starts.txt

```diff
@@ -0,0 +1,186 @@
+	description `	Now, as you enter the shipyard, you feel like this is the start of a new life as a captain.`
+	thumbnail "scene/hai start"
+	system "Fah Soom"
+	planet "Greenwater"
+	date 16 11 3013
+	to display
+		or
+			has "main plot completed"
+			has "Ask the Hai about the Unfettered"
+	on display
+		description "This start is locked for now."
+	to unlock
+		has "main plot completed"
+		# TODO: think about what point of HR should unlock this once it's back in the game, if any
+		# has "event: hai-human resolution announced"
+	conversation "hai intro"
+	account
+		credits 1431000
+		score 400
+		mortgage Mortgage
+			principal 1431000
+			interest 0.003
+			term 730
+	set "species: human"
+	set "hai space start"
+	# need to set it here to not have it trigger alongside the invisible mission
+	set "First Contact: Hai: offered"
+	set "Intro [0]: offered"
+
+
+conversation "hai intro"
+	scene "scene/hai shipyard"
+	`Going to the North Pole already makes you feel as if you're on another world. Though you've met some Hai in the past, you've never been among so many of them in one place; the movement of the crowd in this Hai city far outstrips the equatorial human town where you grew up.`
+	scene "scene/hai bank"
+	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of the pristine ships advertised here.`
+	`	To your disappointment, the only ship you can afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life. However, upon receiving the papers you find out they are in Hai, which you can't make sense of. Probably seeing the confusion on your face, they promptly give you new papers and give you time to read. Only now do you fully realize how risky this is. You might have to cut some costs once you get your hands on the ship.`
+	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "You are accepting a loan in the amount of 1,431,000 credits, due over the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
+	name
+	`	You head out of the bank, but see a notice on your way to buy the Aphid at the shipyard. A retiring captain is selling off his fleet at a reduced price, and one of the ships he's offering is a Grasshopper... maybe you could get a deal on it.`
+	choice
+		`	(Go to the shipyard.)`
+			goto shipyard
+		`	(Go see what the captain has to offer first.)`
+	`	You head to the spaceport and easily find the captain, a Hai, selling one of his Centipedes to another Hai. Thankfully it looks like he has not sold the Grasshopper yet. They happen to be conversing in your language, and you can overhear part of the conversation as the buyer is speaking. "... great that some people still care about helping others make their way to the stars faster, instead of only looking at profit."`
```

Comment:
Looks like I missed something. 
```suggestion
	`	You head to the spaceport and easily find the captain, who is a Hai, selling one of his Centipedes to another Hai. Thankfully, it looks like he has not sold the Grasshopper yet. They happen to be conversing in your language, and you can overhear part of the conversation as the buyer is speaking. "... great that some people still care about helping others make their way to the stars faster, instead of only looking at profit."`
```